gapir/cc/surface.h

```diff
@@ -45,12 +45,7 @@ struct Win32WindowInfo {
 };
 #endif
 
-enum SurfaceType {
-  Unknown,
-  Android,
-  Win32,
-  Xcb,
-};
+enum SurfaceType { Unknown, Android, Win32, Xcb, Ggp };
```

Comment:
I tend to prefer one definition by line, but maybe it's the formatter choice.

Reply:
Done